pom.xml

```diff
@@ -85,8 +85,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>jenkins-war</artifactId>
-      <!--to have access to User.getById-->
-      <version>1.651.2</version>
+      <version>2.138.3</version>
```

Comment:
Matrix authorization plugin requires Jenkins >= 2.138.3 for configuration as code support.